README.md

```diff
@@ -97,3 +190,4 @@ company like no other.
 Contributors:
 
 * Thomas Schoffelen, [@tschoffelen](https://twitter.com/tschoffelen)
+* César Jeanroy, [@cesar3030](https://github.com/cesar3030)
```

Comment:
Is it fine with you if I add myself as a contributor? 😄

Reply:
Of course!

Reply:
Thanks!